engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/LibUtils.java

```diff
@@ -355,21 +355,24 @@ private static void loadNativeLibrary(String path) {
 
     private static LibTorch downloadPyTorch(Platform platform) {
         String version = platform.getVersion();
-        String flavor = platform.getFlavor();
         String classifier = platform.getClassifier();
         String precxx11;
-        if (System.getProperty("os.name").startsWith("Linux")
-                && (Boolean.getBoolean("PYTORCH_PRECXX11")
-                        || Boolean.parseBoolean(Utils.getEnvOrSystemProperty("PYTORCH_PRECXX11"))
-                        || ("aarch64".equals(platform.getOsArch())
-                                && "linux".equals(platform.getOsPrefix())))) {
-            precxx11 = "-precxx11";
+        String flavor = Utils.getEnvOrSystemProperty("PYTORCH_FLAVOR");
+        if (flavor == null || flavor.isEmpty()) {
+            flavor = platform.getFlavor();
+            if (System.getProperty("os.name").startsWith("Linux")
```

Comment:
Is there a reason we're using System property here instead of `platform.getOsPrefix()` ?

Reply:
Cool, that's what i thought just wanted to confirm.